### PR TITLE
Change/api funcs

### PR DESCRIPTION
**Pull request by bakhterets**
separating changes to the API from the "aware datetime" branch

### CODE REVIEW

**Comment by bakhterets** [comment]
image
image

**Comment by Aloento** [comment]
Still Draft

File: app/api/routes.py [hunk 1]
```diff
@@ -129,14 +140,17 @@ def handling_incidents(
...
-            dst_incident,
-            f"{comp_with_attrs} moved from {src_incident.text}"
-        )
+
+        url_d = url_for('web.incident', incident_id=dst_incident.id)
+        url_s = url_for('web.incident', incident_id=src_incident.id)
+        link_s = f"<a href='{url_d}'>{dst_incident.text}</a>"
+        link_d = f"<a href='{url_s}'>{src_incident.text}</a>"
+        update_s = f"{comp_with_attrs} moved to {link_s}"
+        update_d = f"{comp_with_attrs} moved from {link_d}"
```
Comment (vladimirhasko) [inline app/api/routes.py:149]: during testing I don't see the details only the strings moved to and moved from
image

**Comment by vladimirhasko** [comment]
The issue has been detected when the component is split/created to new incident on UI frontend (without API)
then the empty moved to and moved from message appears as seen on screenshots

**Review by Aloento — APPROVED** [review]

**Pull request merged**